frontend/src/component/admin/network/NetworkTrafficUsage/chart-functions.ts

```diff
@@ -45,51 +46,38 @@ export const toTrafficUsageChartData = (
 };
 
 export const toConnectionChartData = (
-    traffic: TrafficUsageDataSegmentedCombinedSchema,
+    traffic: MeteredConnectionsSchema,
 ): { datasets: ChartDatasetType[]; labels: string[] } => {
     const { newRecord, labels } = getLabelsAndRecords(traffic);
-    const datasets = traffic.apiData
-        .filter((apiData) => apiData.apiPath === '/api/client')
```

Comment:
we can remove filtering/sorting/complex mapping because backend returns what we need